composer.json

```diff
@@ -6,18 +6,15 @@
     "license": "BSD-3-Clause",
     "require": {
         "silverstripe/recipe-plugin": "^1",
-        "silverstripe/recipe-cms": "^4.5",
-        "silverstripe/restfulserver": "2.1.x-dev",
-        "silverstripe/versionfeed": "2.0.x-dev"
+        "silverstripe/recipe-cms": "^4@dev",
```

Comment:
As you mentioned on another PR, `@dev` is inert here ;)

Reply:
yeah. could delete it for consistency, though, it's no biggie I reckon

Reply:
fixed